great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -2429,7 +2425,6 @@ def _spark_column_map_condition_value_counts(
 
     result_format = metric_value_kwargs["result_format"]
 
-    filtered = data.filter(F.col("__unexpected") == True).drop(F.col("__unexpected"))
```

Comment:
@NathanFarmer I believe that we can leave the `filtered =` statement where it was (because there could be an exception raised earlier, so no need to have it before).  Would you agree or not?  Thanks.

Reply:
@alexsherstinsky Sure, we can leave this line where it is, but then I would advocate to move lines 2408-2409 down below as well for consistency.

Reply:
@NathanFarmer The way I am seeing it now seems good.  Thank you.